src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -75,24 +75,54 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('iot dps access-policy create') as c:
         c.argument('rights', options_list=['--rights', '-r'], nargs='+',
                    arg_type=get_enum_type(AccessRightsDescription),
-                   help='Access rights for the IoT provisioning service. Use space-separated list for multiple rights.')
+                   help='Access rights for the IoT Hub Device Provisioning Service. '
+                        'Use space-separated list for multiple rights.')
         c.argument('primary_key', help='Primary SAS key value.')
         c.argument('secondary_key', help='Secondary SAS key value.')
 
     with self.argument_context('iot dps access-policy update') as c:
         c.argument('rights', options_list=['--rights', '-r'], nargs='+',
                    arg_type=get_enum_type(AccessRightsDescription),
-                   help='Access rights for the IoT provisioning service. Use space-separated list for multiple rights.')
+                   help='Access rights for the IoT Hub Device Provisioning Service. '
+                        'Use space-separated list for multiple rights.')
+        c.argument('primary_key', help='Primary SAS key value.')
+        c.argument('secondary_key', help='Secondary SAS key value.')
+
+    with self.argument_context('iot dps policy') as c:
+        c.argument('access_policy_name', options_list=['--policy-name', '--name', '-n'],
```

Comment:
Like we were talking about earlier, we can use this as an opportunity to slowly conform to `azure-iot` extension conventions for a consistent user Xp.

`c.argument('access_policy_name', options_list=['--policy-name', '--pn']` or similar and have `--dps-name` support `-n`.